src/main/java/lotto/domain/Profit.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import static lotto.util.LottoConstants.LOTTO_PRICE;
+
+public class Profit {

```

Comment:
이율을 계산하여 반환하는 객체인데 `Profit`은 클래스 이름이 모호한것 같습니다. 

```suggestion
public class ProfitRateCalculator {
    private final static int PERCENT = 100;

    public double calculateTotalProfit(LottoResult lottoResult, LottoCount lottoCount) {
        return (getWinningPrize() / getPurchasedPrice()) * PERCENT;
    }

    private double getPurchasedPrice(LottoCount lottoCount) {
        return lottoCount.getCount() * LOTTO_PRICE.getValue();
    }

    private double getWinningPrize(LottoResult lottoResult) {
        return lottoResult.calculateWinningPrize();
    }
}
```
와 같이 변경하면 의도가 더 명확해질 것 같습니다.